roles/identity-management/manage-rh-sso-identities/README.md

```diff
@@ -47,8 +50,30 @@ This role currently supports the usage of two different tags:
 
 - **create-user:** Specify this tag to create one or more users
```

Comment:
Is there a reason we need to run the creation of users and groups seperately? I'd expect this to run similar to the IdM role where if there are users or groups present, it handles them and if there isn't any, then it doesn't.